projects/plugins/jetpack/tests/e2e/lib/flows/jetpack-connect.js

```diff
@@ -155,22 +152,23 @@ export async function connectThroughJetpackStart( {
 	} );
 
 	// Logs in to WPCOM
-	const login = await LoginPage.visit( page );
-	if ( ! ( await login.isLoggedIn() ) ) {
-		await login.login( wpcomUser );
+	const loginPage = await LoginPage.visit( page );
+	if ( ! ( await loginPage.isLoggedIn() ) ) {
+		await loginPage.login( wpcomUser );
 	}
 
 	const nextUrl = provisionJetpackStartConnection();
 	// sometimes after clicking on Approve button below user being redirected to wp-login page
 	// maybe waiting for a bit will help?
 	await page.waitForTimeout( 10000 );
 
-	await ( await AuthorizePage.visit( page, nextUrl ) ).approve();
-	await ( await PlansPage.init( page ) ).isCurrentPlan( 'business' );
-
-	const siteUrl = getTunnelSiteUrl();
+	// We cannot use AuthorizePage.visit because of the dynamic url
+	await loginPage.goto( nextUrl );
+	const authorizePage = AuthorizePage.init( page );
 
-	await ( await WPLoginPage.visit( page, siteUrl + '/wp-login.php' ) ).login();
+	await authorizePage.approve();
```

Comment:
We probably can follow the same pattern as calls below: `await ( await AuthorizePage.init( page ).approve()` or at least add a missing `await` on line 167

Reply:
Done